### PR TITLE
Improving frontend

### PR DESCRIPTION
**Pull request by ghost**
See #930

Making it look like this: https://user-images.githubusercontent.com/61570792/146810927-ebb2a25c-4dfb-487a-9757-e0c08b277c7e.png

~~My changes currently look like this:~~ (they've been slightly modified actually, see comments)

image

TODO:
- [x] Get base done
- [ ] Determine if the users prefer a light theme or a dark theme so that we can determine what a good default will be
- [ ] Add light theme
- [x] Fix sizing for the sidebar so that it looks more like the concept
- [ ] Make the chosen.js dropdown look more like the concept
- [x] #3026 
- [ ] Remove unused CSS, clean up CSS
- [ ] Re-add the "back to top" button
- [x] Add [this plugin](https://github.com/jekyll/jekyll-seo-tag) so that we don't have to do all the meta tags ourselves.

### CODE REVIEW

**Comment by ritwik12** [comment]
@ConnorAhern This looks good, few things:

- It would be better to keep the default as light mode only, our site looks much better in that.
- We can surely work on the logo, I will create a separate issue for that.
- We can discuss about that in #930

**Comment by ghost** [comment]
Alright. I will update it once I get home.

**Comment by ghost** [comment]
Forgot to mention it, but I have updated it.

**Comment by ghost** [comment]
Oops-
image
Fixing in a second

**Comment by ghost** [comment]
What exactly is going on here...? I'm not sure what it means by "exited with error code 1" - also, I tested it in my browser, and it seems to work.
image

File: _config.yml [hunk 1]
```diff
@@ -27,6 +25,14 @@ exclude:
 
 github: [metadata]
 
+plugins:
+  - jekyll-seo-tag
```
Comment (ritwik12) [inline _config.yml:29]: @ConnorAhern Does this work with gh-pages? That could be the reason for failing. Read [here](https://stackoverflow.com/questions/53638446/jekyll-seo-tag-doesnt-work-within-github-pages)

Comment (ghost) [reply _config.yml:29]: It does work with GH Pages. I have used it before.

According to the StackOverflow post, they just used it wrong.

**Comment by ghost** [comment]
image

**Comment by ritwik12** [comment]
@ConnorAhern Working on this?

**Pull request closed without merging**